test/functional/columns/readonly/columns-readonly.ts

```diff
@@ -34,9 +34,20 @@ describe("columns > readonly functionality", () => {
         const loadedPost = await postRepository.findOne(1);
         expect(loadedPost!.title).to.be.equal("About columns1");
         expect(loadedPost!.text).to.be.equal("Some text about columns1");
-        expect(loadedPost!.authorName).to.be.equal("Umed1");
+        expect(loadedPost!.authorName).to.be.equal("Umed");
 
-    })));
 
+        // then update all its properties and save again
+        post.title = "About columns1";
+        post.text = "Some text about columns1";
+        post.authorName = "Umed1";
```

Comment:
I think this should throw an error. If property is readonly we should get an error on assignment. Just ignoring such situations may lead to bugs in app logic.

Reply:
Any updates here? Reading `tests` code and saw this bug.
`expect(loadedPost!.authorName).to.be.equal("Umed1");`

I'd expect `post.authorName` NOT be updated and get an error or at least a consistent expected behavior.